src/utils/urlJoin.ts

```diff
@@ -1,5 +1,7 @@
-export default function urlJoin(...parts: Array<any>) {
-  return parts.reduce((r, next) => {
+export default function urlJoin(
+  ...parts: Array<string | undefined | null>
```

Comment:
i think these might actually be allowed to include numbers and such

Reply:
just adding `number`s for now